package.json

```
@@ -13,12 +13,15 @@
     "cache:clear": "rm -rf .cache",
     "evals": "npm run build-dom-scripts && tsx evals/index.eval.ts",
     "build-dom-scripts": "tsx lib/dom/genDomScripts.ts",
-    "build": "npm run build-dom-scripts && tsup lib/index.ts --dts",
-    "release": "changeset publish"
+    "build-types": "tsc --emitDeclarationOnly --outDir dist",
+    "build-js": "esbuild lib/index.ts --bundle --platform=node --outdir=dist --format=esm --minify --tree-shaking=true --packages=external",
```

tree shaking is already enabled according to the docs:

```suggestion
    "build-js": "esbuild lib/index.ts --bundle --platform=node --outdir=dist --format=esm --minify --packages=external",
```

https://esbuild.github.io/api/#tree-shaking